File: Lab_loop/t14.cpp
#include <stdio.h>
int main(){
    int mycard = 3, guess;
    do{
        puts("Guess my card: ");
        scanf("%d", &guess);
        if (guess == mycard)
            puts("Good guess!");
        else
            puts("Try again.");
    }while(guess != mycard);
    return 0;
}

File: Lab_loop/t14.java
import java.util.Scanner;
public class t14{
    public static void main(String[] args) {
        int mycard = 3, guess;
        Scanner scan = new Scanner(System.in);
        do{
            System.out.println("Guess my card: ");
            guess = scan.nextInt();
            if (guess == mycard)
                System.out.println("Good guess!");
            else
                System.out.println("Try again.");
        }while(guess != mycard);
        scan.close();
    }
}